src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css

```diff
@@ -121,6 +123,16 @@
 	flex-shrink: 0;
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab.sizing-fixed {
+	min-width: var(--tab-sizing-current-width, 50px);
+	max-width: var(--tab-sizing-current-width, var(--tab-sizing-max-width, 240px));
+	flex: 1 0 0; /* all tabs are evenly sized and grow */
+
+	/* delay, duration, timing function, all apply to both properties */
+	transition: var(--tab-sizing-transition-duration, 0.25s) var(--tab-sizing-transition-duration, 0.25s) linear;
```

Comment:
I think we should only have the transition when mouse is over tabs. Otherwise, you see an animation when just changing the max width setting.

Reply:
The transition is actually visible only when the mouse leaves the tab bar and the tabs are allowed to grow or shrink again. 

When changing the max width setting, I like the transition. If I think of a nice way of disabling the transition for settings change, I'll play with it.

Reply:
I think it's OK with the transition being more immediate now.